test/dlc_tests/container_tests/bin/pytorch_tests/testNVApex

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
+set -e
+
 HOME_DIR=/test
 BIN_DIR=${HOME_DIR}/bin
 
 echo "Testing Nvidia Apex imports"
 
-python -c "import torch; import apex; from apex import amp; from apex.parallel import DistributedDataParallel; from apex import optimizers; from apex.fp16_utils import *"
+python -c "import torch; import apex; from apex import amp; from apex.parallel import DistributedDataParallel; from apex import optimizers; from apex.fp16_utils import *" || exit 1
```

Comment:
shall we just disable this test properly if we are removing apex?

Reply:
It's disabled.